<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CD059" ignored="0" name="S4_SelfFullBillPaidWithGroup" tests="1" failures="1" timestamp="18 Jul 2018 08:39:52 GMT" time="167.004" errors="0">
  <testcase name="s4_SelfFullBillPaidWithGroup" time="159.769" classname="tc_BasicDispatch_SelectPrinter_SampleAcknowledgementByTechnician.S4_SelfFullBillPaidWithGroup">
    <failure type="java.lang.AssertionError" message="***Alert not displayed as expected after bulk entry*** expected [Results Entered Successfully] but found [Enter the Result Value]">
      <![CDATA[java.lang.AssertionError: ***Alert not displayed as expected after bulk entry*** expected [Results Entered Successfully] but found [Enter the Result Value]
	at tc_BasicDispatch_SelectPrinter_SampleAcknowledgementByTechnician.S4_SelfFullBillPaidWithGroup.s4_SelfFullBillPaidWithGroup(S4_SelfFullBillPaidWithGroup.java:140)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- s4_SelfFullBillPaidWithGroup -->
</testsuite> <!-- S4_SelfFullBillPaidWithGroup -->
